data_collection/api/tracking_api.yaml

```diff
@@ -850,8 +850,9 @@ paths:
         - name: ts_v
           in: query
           description: |
-            Version of a library used for tracking.
+            Version of a library used for tracking. [Semantic Versioning](https://semver.org) format required.
```

Comment:
If you want to link description of this term, then why only here? Other YAML files are also mentioning it.

Reply:
I've added links to each Semantic Versioning occurrence.